docs/modules/ROOT/pages/how-tos/metadata/provenance.adoc

```diff
@@ -1,4 +1,142 @@
-= Downloading the SLSA provenance
+= Downloading your SLSA provenance
```

Comment:
Why this change? It is the provenance for some specific artifact, not necessarily related to the user